pkg/operator/ceph/object/rgw.go

```diff
@@ -25,8 +25,10 @@ import (
 	"github.com/rook/rook/pkg/clusterd"
```

Comment:
All these changes seem to focus on deployments. What about the daemonsets?

Reply:
Daemonsets are gone with: https://github.com/rook/rook/pull/3283